### PR TITLE
[IMP] inventory: fix and update lots

### PR DESCRIPTION
**Pull request by brse-odoo**
This PR is targeting Odoo project task [#3215736](https://www.odoo.com/web#id=3215736&cids=3&menu_id=4720&action=333&active_id=3835&model=project.task&view_type=form) to make a small fix on the lots doc.

Made small fix, and performed complete rewrite to the document for 14.0 to be fwd-ported (targeted revision for 15.0 and 16.0 to follow).

- Completely rewrote the .rst file and added example flows
- Replaced and updated all screenshots
- Doc is more robust and comprehensive than before

### CODE REVIEW

**Comment by brse-odoo** [comment]
Hi @jero-odoo and @larm-odoo! Whichever of you two is available to do so, this PR is ready for round one of peer review. Thank you in advance!

File: content/applications/inventory_and_mrp/inventory/management/lots_serial_numbers/lots.rst
Comment (jero-odoo) [inline content/applications/inventory_and_mrp/inventory/management/lots_serial_numbers/lots.rst:201]: Instead of describing the benefits of lots overall, this can be one sentence explaining the traceability report. "The traceability report shows x, y, and z". Otherwise it sounds a bit too much like an introduction to a new document.

**Review by jero-odoo — COMMENTED** [review]
@brse-odoo All set! Let me know if you have any questions 👍

File: content/applications/inventory_and_mrp/inventory/management/lots_serial_numbers/lots.rst
Comment (brse-odoo) [reply content/applications/inventory_and_mrp/inventory/management/lots_serial_numbers/lots.rst:58]: I used a `:guilabel:` here since technically it's UI that you interact with, I'll leave this up to final review

File: content/applications/inventory_and_mrp/inventory/management/lots_serial_numbers/lots.rst
Comment (brse-odoo) [reply content/applications/inventory_and_mrp/inventory/management/lots_serial_numbers/lots.rst:70]: Not always - specified this ahead of this sentence to clear up

File: content/applications/inventory_and_mrp/inventory/management/lots_serial_numbers/lots.rst
Comment (brse-odoo) [reply content/applications/inventory_and_mrp/inventory/management/lots_serial_numbers/lots.rst:155]: Good call

**Comment by brse-odoo** [comment]
Hi @odoo/us-doc-review, this PR is ready for a content review when one of you has a chance. Thank you!

File: content/applications/inventory_and_mrp/inventory/management/lots_serial_numbers/lots.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/management/lots_serial_numbers/lots.rst:109]: reformat suggestion / rephrasing

```suggestion
Assign lot numbers manually
~~~~~~~~~~~~~~~~~~~~~~~~~~~
To assign lot numbers manually, click :guilabel:`Add a line` from the :guilabel:`Detailed Operations` pop-up, and first choose the location where the product will be stored under the :guilabel:`To` column.

Then, type a new :guilabel:`Lot Number Name`, and set the :guilabel:`Done` quantity in the appropriate columns. 
```

File: content/applications/inventory_and_mrp/inventory/management/lots_serial_numbers/lots.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/management/lots_serial_numbers/lots.rst:125]: reformat suggestion / rephrase

```suggestion
Copy/paste lot numbers from an Excel file
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
To copy and paste lot numbers from an existing *Excel* file, first populate an *Excel* spreadsheet with all of the lot numbers received from the supplier (or manually chosen upon receipt). Then, copy and paste them in the :guilabel:`Lot/Serial Number Name` column. Odoo will automatically create the necessary number of lines based on the amount of numbers pasted in the column.

From here, the :guilabel:`To` locations and :guilabel:`Done` quantities can be manually entered in each of the serial number lines.
```

File: content/applications/inventory_and_mrp/inventory/management/lots_serial_numbers/lots.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/management/lots_serial_numbers/lots.rst:135]: slight rephrasing

```suggestion
Once all product quantities have been assigned a lot number, click the :guilabel:`Confirm` button to close
the pop-up. Then, click :guilabel:`Validate`.

A :guilabel:`Traceability` smart button appears upon validating the receipt. Click the :guilabel:`Traceability` smart button to see the updated :guilabel:`Traceability Report`, which includes: a :guilabel:`Reference` document, the
:guilabel:`Product` being traced, the :guilabel:`Lot/Serial #`, and more.
```

File: content/applications/inventory_and_mrp/inventory/management/lots_serial_numbers/lots.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/management/lots_serial_numbers/lots.rst:144]: rephrasing

```suggestion
With Odoo *Inventory*, lot numbers can be assigned to *outgoing* goods directly from a sales order.

To create a :abbr:`SO (Sales Order)`, navigate to the *Sales* app, and click the :guilabel:`Create` button. Doing so, reveals a new, blank quotation form. On this blank quotation form, fill out the necessary information, by adding a :guilabel:`Customer`, and adding products to the :guilabel:`Product` lines (in the :guilabel:`Order Lines` tab), by clicking :guilabel:`Add a product`. 

Then, choose the desired quantity to sell by changing the number in the :guilabel:`Quantity` column.
```

File: content/applications/inventory_and_mrp/inventory/management/lots_serial_numbers/lots.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/management/lots_serial_numbers/lots.rst:148]: rephrasing

```suggestion
Once the quotation has been filled out, click the :guilabel:`Confirm` button to confirm the quotation. When the quotation is confirmed, the quotation becomes a :abbr:`SO (Sales Order)`, and a :guilabel:`Delivery` smart button appears.

Click the :guilabel:`Delivery` smart button to be view to the :guilabel:`Warehouse Receipt Form` for that specific :abbr:`SO (Sales Order)`.
```

File: content/applications/inventory_and_mrp/inventory/management/lots_serial_numbers/lots.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/management/lots_serial_numbers/lots.rst:155]: rephrasing

```suggestion
From here, click the :guilabel:`Additional Options` menu, represented by a `hamburger` icon (four vertical lines, located to the right of the :guilabel:`Unit of Measure` column in the :guilabel:`Operations` tab). Clicking that icon reveals a :guilabel:`Detailed Operations` pop-up.

In the pop-up, a :guilabel:`Lot/Serial Number` will be chosen by default, with the full :guilabel:`Reserved` quantity taken from that specific lot (if there is enough stock in that particular lot).

If there is insufficient stock in that lot, or if partial quantities of the :guilabel:`Demand` should be taken from multiple lots, change the quantity in the :guilabel:`Done` column to only include that specific part of the total quantity.
```

File: content/applications/inventory_and_mrp/inventory/management/lots_serial_numbers/lots.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/management/lots_serial_numbers/lots.rst:161]: slight grammatical adjustment

```suggestion
    The lot automatically chosen for delivery orders varies, depending on the selected removal
    strategy (:abbr:`FIFO (First In, First Out)`, :abbr:`LIFO (Last In, First Out)`, or
    :abbr:`FEFO (First Expiry, First Out)`). It will also depend on the quantity ordered, and if
    there is enough quantity in one lot to fulfill the order.
```

File: content/applications/inventory_and_mrp/inventory/management/lots_serial_numbers/lots.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/management/lots_serial_numbers/lots.rst:168]: minor adjustment

```suggestion
pop-up. Lastly, click the :guilabel:`Validate` button to deliver the products.
```

File: content/applications/inventory_and_mrp/inventory/management/lots_serial_numbers/lots.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/management/lots_serial_numbers/lots.rst:178]: slight rephrasing

```suggestion
Upon validating the delivery order, a :guilabel:`Traceability` smart button appears. Click the :guilabel:`Traceability` smart button to see the updated :guilabel:`Traceability Report`, which includes: a :guilabel:`Reference` document, the :guilabel:`Product` being traced, the :guilabel:`Date`, and the :guilabel:`Lot/Serial #` assigned.

The :guilabel:`Traceability Report` can also include a :guilabel:`Reference` receipt from the previous purchase order, if the product quantities shared the same lot number.
```

File: content/applications/inventory_and_mrp/inventory/management/lots_serial_numbers/lots.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/management/lots_serial_numbers/lots.rst:185]: minor adjustment

```suggestion
In Odoo, the creation of new lots is only allowed upon *receiving* products from a purchase order, by default. *Existing* lot numbers cannot be used. 

For sales orders, the opposite is true: new lot numbers cannot be created on the delivery order, only existing lot numbers can be used.
```

File: content/applications/inventory_and_mrp/inventory/management/lots_serial_numbers/lots.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/management/lots_serial_numbers/lots.rst:193]: rephrasing/reformatted

```suggestion
To change the ability to use new (or existing) lot numbers on any operation type, go to
:menuselection:`Inventory app --> Configuration --> Operations Types`, and select the desired
:guilabel:`Operation Type`. 

For :guilabel:`Receipts`, found on the :guilabel:`Operations Types` page, the :guilabel:`Use Existing Lots/Serial Numbers` option can be enabled, by clicking :guilabel:`Edit`, and then clicking the checkbox beside the :guilabel:`Use Existing Lots/Serial Numbers` option (in the :guilabel:`Traceability` section). Lastly, click the :guilabel:`Save` button to save the changes. 

For :guilabel:`Delivery Orders`, the :guilabel:`Create New Lots/Serial Numbers` option can be enabled, by clicking :guilabel:`Edit`, and clicking the checkbox beside the :guilabel:`Create New Lots/Serial Numbers` option. Be sure to click the :guilabel:`Save` button to save all changes.
```

File: content/applications/inventory_and_mrp/inventory/management/lots_serial_numbers/lots.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/management/lots_serial_numbers/lots.rst:201]: would a :guilabel: be necessary for Receipts in this instance? Might be worth looking into...

File: content/applications/inventory_and_mrp/inventory/management/lots_serial_numbers/lots.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/management/lots_serial_numbers/lots.rst:207]: removed semi-colons

```suggestion
product: where (and when) it came from, where it was stored, and who (and when) it went to.
```

File: content/applications/inventory_and_mrp/inventory/management/lots_serial_numbers/lots.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/management/lots_serial_numbers/lots.rst:212]: slight adjustment

```suggestion
To see the full traceability of a product, or group by lots, go to
:menuselection:`Inventory app --> Products --> Lots/Serial Numbers.` Doing so reveals the :guilabel:`Lots/Serial Numbers` page.

From here, products with lot numbers assigned to them will be listed by default, and can be expanded to show the lot number(s) those products have assigned to them.
```

**Review by ksc-odoo — COMMENTED** [review]
@brse-odoo Just finished reviewing this doc - looks great! If you have any questions, don't hesitate to reach out. Thanks!

**Review by StraubCreative — CHANGES_REQUESTED** [review]
Hi @brse-odoo 
Please fix the 100th char line break issues and tag me again for review.
Thanks

**Review by StraubCreative — CHANGES_REQUESTED** [review]
Still seeing 100th char issues here, as well.
cc: @samueljlieber

File: content/applications/inventory_and_mrp/inventory/management/lots_serial_numbers/lots.rst
Comment (samueljlieber) [inline content/applications/inventory_and_mrp/inventory/management/lots_serial_numbers/lots.rst:24]: The active instruction hasnt started yet here, so we can italicize *Lots & Serial Numbers*
```suggestion
To track products using lots, the *Lots & Serial Numbers* feature must be enabled. Go to
```

File: content/applications/inventory_and_mrp/inventory/management/lots_serial_numbers/lots.rst
Comment (samueljlieber) [inline content/applications/inventory_and_mrp/inventory/management/lots_serial_numbers/lots.rst:89]: Directing the user to the Inventory app
```suggestion
to the product form in the :menuselection:`Inventory` app, by going to :menuselection:`Products -->
Products`, and selecting the product to which this newly-created lot number was just assigned.
```

File: content/applications/inventory_and_mrp/inventory/management/lots_serial_numbers/lots.rst
Comment (samueljlieber) [inline content/applications/inventory_and_mrp/inventory/management/lots_serial_numbers/lots.rst:134]: The active element, the icon, should be in a guilabel and I believe it would be appropriate to put the icon descriptor in the parenthesis
```suggestion
From here, click the :guilabel:`Additional Options` menu, represented by a :guilabel:`hamburger
(four horizontal lines)` icon, located to the right of the :guilabel:`Unit of Measure` column in the
:guilabel:`Operations` tab. Clicking that icon reveals a :guilabel:`Detailed Operations` pop-up.
```

File: content/applications/inventory_and_mrp/inventory/management/lots_serial_numbers/lots.rst
Comment (samueljlieber) [inline content/applications/inventory_and_mrp/inventory/management/lots_serial_numbers/lots.rst:152]: These _can_ be but don't _have_ to be indented to align with the bullet point since they are in context of it - same with the bullet point below

File: content/applications/inventory_and_mrp/inventory/management/lots_serial_numbers/lots.rst
Comment (samueljlieber) [inline content/applications/inventory_and_mrp/inventory/management/lots_serial_numbers/lots.rst:244]: Its best to use menuselection for a page name
```suggestion
For :guilabel:`Receipts`, found on the :menuselection:`Operations Types` page, the :guilabel:`Use
```

File: content/applications/inventory_and_mrp/inventory/management/lots_serial_numbers/lots.rst
Comment (samueljlieber) [inline content/applications/inventory_and_mrp/inventory/management/lots_serial_numbers/lots.rst:268]: Its best to use menuselection for a page name
```suggestion
--> Products --> Lots/Serial Numbers`. Doing so reveals the :menuselection:`Lots/Serial Numbers` page.
```

**Review by samueljlieber — CHANGES_REQUESTED** [review]
Hi @brse-odoo I have just a couple technical changes throughout the doc, please tag me once more for a review after these changes have been made. Thank you!

**Comment by brse-odoo** [comment]
@samueljlieber I've implemented the technical changes you suggested and pushed up the revisions. Back to you -- thank you!

File: content/applications/inventory_and_mrp/inventory/management/lots_serial_numbers/lots.rst
Comment (samueljlieber) [inline content/applications/inventory_and_mrp/inventory/management/lots_serial_numbers/lots.rst:134]: closing parenthesis with no opener 😉
```suggestion
:guilabel:`Operations` tab. Clicking that icon reveals a :guilabel:`Detailed Operations` pop-up.
```

File: content/applications/inventory_and_mrp/inventory/management/lots_serial_numbers/lots.rst
Comment (samueljlieber) [inline content/applications/inventory_and_mrp/inventory/management/lots_serial_numbers/lots.rst:195]: Icon formatting again
```suggestion
From here, click the :guilabel:`Additional Options` menu, represented by a :guilabel:`hamburger (four horizontal lines)` icon , located to the right of the :guilabel:`Unit of Measure` column in the :guilabel:`Operations` tab. Clicking that icon reveals a :guilabel:`Detailed Operations` pop-up.
```

**Review by samueljlieber — APPROVED** [review]
Hi @brse-odoo just two technical corrections I found, please see them below. I am pushing these up as a commit for you after this review since they are minor and I want to get this PR over to final review! Good work with this PR 🙂

File: content/applications/inventory_and_mrp/inventory/management/lots_serial_numbers/lots.rst
Comment (StraubCreative) [inline content/applications/inventory_and_mrp/inventory/management/lots_serial_numbers/lots.rst:46]: Cutting down on wordiness
```suggestion
:guilabel:`Inventory` tab. In the :guilabel:`Traceability` section, click :guilabel:`By Lots`. Then,
:guilabel:`Save` the changes. Existing or new lot numbers can now be assigned to newly-received or
manufactured batches of this product.
```

File: content/applications/inventory_and_mrp/inventory/management/lots_serial_numbers/lots.rst
Comment (StraubCreative) [inline content/applications/inventory_and_mrp/inventory/management/lots_serial_numbers/lots.rst:92]:
```suggestion
lot number. When additional quantities of this product is received or manufactured, this new lot
```

File: content/applications/inventory_and_mrp/inventory/management/lots_serial_numbers/lots.rst
Comment (StraubCreative) [inline content/applications/inventory_and_mrp/inventory/management/lots_serial_numbers/lots.rst:107]: grammar
```suggestion
To create a :abbr:`PO (purchase order)`, go to :menuselection:`Purchase app --> Create`. Doing
```

File: content/applications/inventory_and_mrp/inventory/management/lots_serial_numbers/lots.rst
Comment (StraubCreative) [inline content/applications/inventory_and_mrp/inventory/management/lots_serial_numbers/lots.rst:112]: redundant
```suggestion
:guilabel:`Add a product`.
```

File: content/applications/inventory_and_mrp/inventory/management/lots_serial_numbers/lots.rst
Comment (StraubCreative) [inline content/applications/inventory_and_mrp/inventory/management/lots_serial_numbers/lots.rst:133]: Another way to write this
```suggestion
From here, click the :guilabel:`Additional Options` menu, represented by a :guilabel:`≣ 
(hamburger)` icon, located to the right of the :guilabel:`Unit of Measure` column in the
```

File: content/applications/inventory_and_mrp/inventory/management/lots_serial_numbers/lots.rst
Comment (StraubCreative) [inline content/applications/inventory_and_mrp/inventory/management/lots_serial_numbers/lots.rst:139]: Grammar is off between these two paragraphs.
"To do this" ... to do what exactly?

```suggestion
In this pop-up, configure a number of different fields, including the assignation of a lot number,
under the :guilabel:`Lot/Serial Number Name` column, located at the bottom of the pop-up.

Assign lot numbers either **manually** or by **copying/pasting from a spreadsheet**.
```

**Review by StraubCreative — APPROVED** [review]
Hi @brse-odoo 
Well-written doc 👏 
Approving with comments around grammar and wordiness.

@robodoo delegate+

**Comment by brse-odoo** [comment]
@robodoo r+

**Pull request closed without merging**
